cli-commands/commands.js

```diff
@@ -43,12 +43,23 @@ const commands = [
 			yargs.positional('gasPrice', {
 				describe: 'The price of gas in wei - default is 20000000000',
 				type: 'number'
-			})
+			});
 
 			yargs.positional('gasLimit', {
 				describe: 'The block gas limit default is 0x6691b7',
 				type: 'number'
-			})
+			});
+
+			yargs.positional('mnemonic', {
+				describe: 'Pass mnemonic to generate account',
+				type: 'string'
+			});
+
+			yargs.positional('generate', {
```

Comment:
generate is not a very good name for "number of accounts to generate" - lets find a better name

Reply:
```count``` ?

Reply:
Yes, or accounts-count